docs/installation.md

```diff
@@ -127,9 +127,12 @@ __Install SDK Wheel__
 
 __Start the Xailient Daemon which activates your license__
 
-```bash
-(env) $ sudo python3 -m xailient.install
 ```
+(env) $ sudo /<Xailient Install Location>/xailient/scripts/xailient-install
```

Comment:
We don't need to know the install location to run the Xailient-install script anymore.
`sudo python3 -m xailient.install` automatically finds Xailient-install and runs it.
This was part of the some changes I made to the SDK to make it easier to install. I always thought since even before I joined Xailient that having to `cd` into the python package's install dir was awkward. Hence my desire to simplify this. Literally users can now just do `pip3 install <path to .whl file> && python3 -m xailient.install` and everything is done. The daemon gets installed and started, etc.

Reply:
Oops, sorry about that. Is this in the SDK 1.0.7?
This is awesome. So much better from the user's perspective.
I'll change this back. Thanks :)